untyped-lambda-calculus/article.md

```diff
@@ -148,19 +162,6 @@ A practically significant variation on call-by-name is _call-by-need_, which is,
 Call-by-need behaves exactly like call-by-name, except that reduction steps that would duplicate computations, don't.
 This is achieved by sharing the computation corresponding to an argument in all places where it is used.
 
-<p>
-<details>
-<summary>Haskell and call by need</summary>
-
-In theory, Haskell strives to implement call-by-need.
-In practice, it is a little more complicated.
-The reason is simple -- the optimizer can make non-obvious transformations, which could either remove some duplicated computations or vice versa.
-
-***
-
-</details>
-</p>
-
```

Comment:
@NaeosPsy, ?

Did you mean to move it somewhere, or did you decide this is irrelevant?

Reply:
I did decide it is not very relevant. It also looked very ugly when rendered so I thought it would be better if it was gone.

Reply:
Well, it's arguably an important caveat to keep in mind, but I guess it's fine to remove, as it is only tangentially related to the topic at hand. :ok_hand: